src/handlers/tokenLocksHandler.ts

```diff
@@ -28,8 +28,8 @@ const tokenUnlockResponse = (transaction) => ({
   hash: transaction.hash,
   amount: transaction.amount,
   source: transaction.source_addr,
+  tokenLockRef: transaction.lock_reference_hash,
```

Comment:
We also need the global snapshot ordinal that this tokenUnlock was included in here.